internal/dev_server/model/project.go

```diff
@@ -36,6 +38,7 @@ func CreateProject(ctx context.Context, projectKey, sourceEnvironmentKey string,
 	project.FlagState = flagsState
 	project.LastSyncTime = time.Now()
 
+	store := StoreFromContext(ctx)
```

Comment:
Just a cleanup - bundled the project variable initialization together into one statement and also moved the store fetch closer to where the store is used.